packages/hub/services/api-router.ts

```diff
@@ -153,6 +155,7 @@ export default class APIRouter {
     apiSubrouter.get('/wyre-prices', parseBody, wyrePricesRoute.get);
     apiSubrouter.get('/data-integrity-checks/:check_name', parseBody, dataIntegrityChecksRoute.get);
     apiSubrouter.get('/gas-station/:chain_id', gasStationRoute.get);
+    apiSubrouter.post('/gas-estimation', parseBody, gasEstimationRoute.get);
```

Comment:
Post or get?

Reply:
I renamed the function name to `gasEstimationRoute.estimate`